packages/api/lib/schemas.js

```diff
@@ -0,0 +1,753 @@
+'use strict';
+
+module.exports.accessToken = {
```

Comment:
Weird, why is this showing up as an addition, we'd already started using this in feature 🤔